src/Build/Logging/FileLogger.cs

```diff
@@ -269,7 +274,7 @@ private void ApplyFileLoggerParameter(string parameterName, string parameterValu
         /// <summary>
         /// Encoding for the output. Defaults to UTF-8.
         /// </summary>
-        private Encoding _encoding = Encoding.UTF8;
+        private Encoding _encoding = new UTF8Encoding(false);
```

Comment:
You don't need to pass false because the default ctor of UTF8Encoding passes false already:
http://referencesource.microsoft.com/#mscorlib/system/text/utf8encoding.cs,65